### DISCUSSION TITLE
no logs in SOC or Kabana after soup upgrade

### DISCUSSION THREAD

**Discussion by Xboarder56** [Security-Onion-Solutions/securityonion#6490] [Unsupported Versions]
I performed an upgrade on my standalone instance but I'm not getting any logs afterwards. I have verified they are coming in because tcpdump shows syslog data and network data but the logs are not showing in SOC or Kibana. 

Security Onion version as seen in the lower left corner of SOC and in /etc/soversion.
- 2.3.90

Is this a cloud deployment or on-prem?
- on prem stand alone

Did you install from our Security Onion ISO image or did you perform a network installation?
- Security Onion ISO

How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Standalone

Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Don't believe its related to the span port. tcpdump shows the logs coming in. 

Does so-status show all services running?
- Yes

```
Checking Docker status

    Docker ---------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng -------------------------------------------------- [ OK ]
    so-cortex ------------------------------------------------------- [ OK ]
    so-curator ------------------------------------------------------ [ OK ]
    so-dockerregistry ----------------------------------------------- [ OK ]
    so-elastalert --------------------------------------------------- [ OK ]
    so-elasticsearch ------------------------------------------------ [ OK ]
    so-filebeat ----------------------------------------------------- [ OK ]
    so-fleet -------------------------------------------------------- [ OK ]
    so-grafana ------------------------------------------------------ [ OK ]
    so-idstools ----------------------------------------------------- [ OK ]
    so-influxdb ----------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------- [ OK ]
    so-kratos ------------------------------------------------------- [ OK ]
    so-logstash ----------------------------------------------------- [ OK ]
    so-mysql -------------------------------------------------------- [ OK ]
    so-nginx -------------------------------------------------------- [ OK ]
    so-playbook ----------------------------------------------------- [ OK ]
    so-redis -------------------------------------------------------- [ OK ]
    so-sensoroni ---------------------------------------------------- [ OK ]
    so-soc ---------------------------------------------------------- [ OK ]
    so-soctopus ----------------------------------------------------- [ OK ]
    so-steno -------------------------------------------------------- [ OK ]
    so-strelka-backend ---------------------------------------------- [ OK ]
    so-strelka-coordinator ------------------------------------------ [ OK ]
    so-strelka-filestream ------------------------------------------- [ OK ]
    so-strelka-frontend --------------------------------------------- [ OK ]
    so-strelka-gatekeeper ------------------------------------------- [ OK ]
    so-strelka-manager ---------------------------------------------- [ OK ]
    so-suricata ----------------------------------------------------- [ OK ]
    so-telegraf ----------------------------------------------------- [ OK ]
    so-thehive ------------------------------------------------------ [ OK ]
    so-thehive-es --------------------------------------------------- [ OK ]
    so-wazuh -------------------------------------------------------- [ OK ]
    so-zeek --------------------------------------------------------- [ OK ]
 ```

Do you get any failures when you run sudo salt-call state.highstate?
- Yes, however I don't believe it's related

`Succeeded: 682 (changed=31)
Failed:      1`

```
ID: prune_images
    Function: cmd.run
        Name: so-docker-prune
      Result: False
     Comment: Command "so-docker-prune" run
     Started: 17:43:25.121747
    Duration: 266.914 ms
     Changes:
              ----------
              pid:
                  25226
              retcode:
                  1
              stderr:
                  so-velociraptor: Invalid version: 'testing'
              stdout:
                  Pruning old containers
```

Does the SOC Grid page show any failures?
- No but zero EPS

Explain your issue. For example: Installation fails when I select this series of options...
- zero logs in kibana or soc (WAZUH, ZEEK, Syslog, Sysmon)

Additional Details:

`[root@vision so]# sudo so-redis-count
249195`

```[root@vision /]# df -h
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  16G     0   16G   0% /dev
tmpfs                     16G  340K   16G   1% /dev/shm
tmpfs                     16G  196M   16G   2% /run
tmpfs                     16G     0   16G   0% /sys/fs/cgroup
/dev/mapper/system-root  293G   39G  255G  14% /
/dev/sda2                497M  169M  329M  34% /boot
/dev/sda1               1022M   12M 1011M   2% /boot/efi
/dev/mapper/system-nsm   720G  637G   83G  89% /nsm
/dev/mapper/system-tmp   2.0G   60M  1.9G   3% /tmp```

**Comment by TOoSmOotH** [comment]
I would start by checking the elasticsearch logs.

**Comment by weslambert** [comment]
> I performed an upgrade on my standalone instance but I'm not getting any logs afterwards. I have verified they are coming in because tcpdump shows syslog data and network data but the logs are not showing in SOC or Kibana.
> 
> Security Onion version as seen in the lower left corner of SOC and in /etc/soversion.
> 
> * 2.3.90
> 
> Is this a cloud deployment or on-prem?
> 
> * on prem stand alone
> 
> Did you install from our Security Onion ISO image or did you perform a network installation?
> 
> * Security Onion ISO
> 
> How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
> 
> * Standalone
> 
> Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
> 
> * Don't believe its related to the span port. tcpdump shows the logs coming in.
> 
> Does so-status show all services running?
> 
> * Yes
> 
> ```
> Checking Docker status
> 
>     Docker ---------------------------------------------------------- [ OK ]
> 
> Checking container statuses
> 
>     so-aptcacherng -------------------------------------------------- [ OK ]
>     so-cortex ------------------------------------------------------- [ OK ]
>     so-curator ------------------------------------------------------ [ OK ]
>     so-dockerregistry ----------------------------------------------- [ OK ]
>     so-elastalert --------------------------------------------------- [ OK ]
>     so-elasticsearch ------------------------------------------------ [ OK ]
>     so-filebeat ----------------------------------------------------- [ OK ]
>     so-fleet -------------------------------------------------------- [ OK ]
>     so-grafana ------------------------------------------------------ [ OK ]
>     so-idstools ----------------------------------------------------- [ OK ]
>     so-influxdb ----------------------------------------------------- [ OK ]
>     so-kibana ------------------------------------------------------- [ OK ]
>     so-kratos ------------------------------------------------------- [ OK ]
>     so-logstash ----------------------------------------------------- [ OK ]
>     so-mysql -------------------------------------------------------- [ OK ]
>     so-nginx -------------------------------------------------------- [ OK ]
>     so-playbook ----------------------------------------------------- [ OK ]
>     so-redis -------------------------------------------------------- [ OK ]
>     so-sensoroni ---------------------------------------------------- [ OK ]
>     so-soc ---------------------------------------------------------- [ OK ]
>     so-soctopus ----------------------------------------------------- [ OK ]
>     so-steno -------------------------------------------------------- [ OK ]
>     so-strelka-backend ---------------------------------------------- [ OK ]
>     so-strelka-coordinator ------------------------------------------ [ OK ]
>     so-strelka-filestream ------------------------------------------- [ OK ]
>     so-strelka-frontend --------------------------------------------- [ OK ]
>     so-strelka-gatekeeper ------------------------------------------- [ OK ]
>     so-strelka-manager ---------------------------------------------- [ OK ]
>     so-suricata ----------------------------------------------------- [ OK ]
>     so-telegraf ----------------------------------------------------- [ OK ]
>     so-thehive ------------------------------------------------------ [ OK ]
>     so-thehive-es --------------------------------------------------- [ OK ]
>     so-wazuh -------------------------------------------------------- [ OK ]
>     so-zeek --------------------------------------------------------- [ OK ]
> ```
> 
> Do you get any failures when you run sudo salt-call state.highstate?
> 
> * Yes, however I don't believe it's related
> 
> `Succeeded: 682 (changed=31) Failed: 1`
> 
> ```
> ID: prune_images
>     Function: cmd.run
>         Name: so-docker-prune
>       Result: False
>      Comment: Command "so-docker-prune" run
>      Started: 17:43:25.121747
>     Duration: 266.914 ms
>      Changes:
>               ----------
>               pid:
>                   25226
>               retcode:
>                   1
>               stderr:
>                   so-velociraptor: Invalid version: 'testing'
>               stdout:
>                   Pruning old containers
> ```
> 
> Does the SOC Grid page show any failures?
> 
> * No but zero EPS
> 
> Explain your issue. For example: Installation fails when I select this series of options...
> 
> * zero logs in kibana or soc (WAZUH, ZEEK, Syslog, Sysmon)
> 
> Additional Details:
> 
> `[root@vision so]# sudo so-redis-count 249195`
> 
> ```
> Filesystem               Size  Used Avail Use% Mounted on
> devtmpfs                  16G     0   16G   0% /dev
> tmpfs                     16G  340K   16G   1% /dev/shm
> tmpfs                     16G  196M   16G   2% /run
> tmpfs                     16G     0   16G   0% /sys/fs/cgroup
> /dev/mapper/system-root  293G   39G  255G  14% /
> /dev/sda2                497M  169M  329M  34% /boot
> /dev/sda1               1022M   12M 1011M   2% /boot/efi
> /dev/mapper/system-nsm   720G  637G   83G  89% /nsm
> /dev/mapper/system-tmp   2.0G   60M  1.9G   3% /tmp```
> ```

You might also try checking the Logstash logs to see if a pipeline failed or other clues. Also try running  `so-logstash-pipeline-stats search` to see if you notice any events being output to Elasticsearch.
